index.html

```diff
@@ -161,7 +162,7 @@ <h2>Definitions</h2>
       <p>
         A <dfn>do-not-sell-or-share interaction</dfn> is an interaction with a website in which the
         person is requesting that their data not be sold to or shared with any party other than the
-        one the person intends to interact with, or to have their data used for cross-site ad targeting,
+        one the person intends to interact with, or to have their data used for [=cross-organization ad targeting=],
```

Comment:
```suggestion
        one the person intends to interact with, or to have their data used for [=cross-site ad targeting=],
```
s/organization/party/